server/client.py

```diff
@@ -155,3 +160,74 @@ def send_message_to(self, msg):
 
         # send msg
         ONLINE_USERS[to].send(PayloadMessage(msg.cmd, [self.user.name], msg.payload))
+
+    @register_cmd(CMD_CREATE_CHAT)
+    @login_required
+    def create_chat(self, msg):
+        """
+        Create new chat and send new chat id back to the client
+        """
+        chat_name = msg.payload
```

Comment:
If I get it right, you're using PayloadMessage instead of NormalMessage to request chat creation. Why?

Reply:
Chant name could have spaces. Then we parse message we use space as a split sequence.